packages/esm-patient-common-lib/src/types/test-results.ts

```diff
@@ -32,3 +30,15 @@ export interface PatientData {
     uuid: string;
   };
 }
+
+export enum OBSERVATION_INTERPRETATION {
```

Comment:
```
ERROR in ../esm-patient-common-lib/src/types/test-results.ts
Module build failed (from ../../node_modules/babel-loader/lib/index.js):
SyntaxError: /home/runner/work/openmrs-esm-patient-chart/openmrs-esm-patient-chart/packages/esm-patient-common-lib/src/types/test-results.ts: 'const' enums are not supported.
  32 | }
  33 |
> 34 | export const enum OBSERVATION_INTERPRETATION {
     |        ^
  35 |   'NORMAL',
  36 |
  37 |   'HIGH',
```
What's the best way here?

Reply:
I think this may be an issue with Babel (for help this they even created a plugin: https://www.npmjs.com/package/babel-plugin-const-enum).

Personally, I'd always go with types. They are more "honest" (as in transparent) and are usually sufficient.

```ts
export type OBSERVATION_INTERPRETATION  = 'foo' | 'bar' | ...;
```